src/blob_storage.h

```diff
@@ -79,6 +100,12 @@ class BlobStorage {
   std::shared_ptr<BlobFileCache> file_cache_;
 
   std::vector<GCScore> gc_score_;
+
+  std::list<std::pair<uint64_t, SequenceNumber>> obsolete_files_;
+  // It is marked when the column family handle is destroyed, indicating this
+  // structure can be removed after all the blob files of this column family are
```

Comment:
the comment sounds misleading. It only means the in-memory data structure can be destroyed. Physical files may still be kept.